backend/cmd/start.go

```diff
@@ -512,12 +518,14 @@ func flagSet(server bool) *pflag.FlagSet {
 		flagSet.String(flagAPIListenAddress, viper.GetString(flagAPIListenAddress), "address to listen on for api traffic")
 		flagSet.Int64(flagAPIRequestLimit, viper.GetInt64(flagAPIRequestLimit), "maximum API request body size, in bytes")
 		flagSet.String(flagAPIURL, viper.GetString(flagAPIURL), "url of the api to connect to")
+		flagSet.Int64(flagAPIWriteTimeout, viper.GetInt64(flagAPIWriteTimeout), "maximum duration before timing out writes of responses")
```

Comment:
As the others have mentioned, let's use `viper.GetDuration()` here instead and update the types surrounding this flag to `time.Duration`.